spec/support/schemas/my_health/prescriptions/v1/prescription_list_filtered_with_pagination.json

```diff
@@ -0,0 +1,50 @@
+{
+  "$schema" : "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "required": ["data", "meta"],
+  "properties": {
+    "data": {
+      "type": "array",
+      "minItems": 1,
+      "uniqueItems": true,
+      "items": {
+        "$ref": "prescription_details.json"
+      }
+    },
+    "meta": {
+      "type": "object",
+      "required": ["updated_at", "failed_station_list", "sort"],
+      "properties": {
+        "updated_at": { "type": "string" },
+        "failed_station_list": { "type": "string" },
+                "filter": {
+          "type": "object"
+        },
+        "sort": {
+          "type": "object"
+        },
+        "pagination": {
+          "type": "object",
+          "required": ["current_page", "per_page", "total_pages", "total_entries"],
+          "properties": {
+            "current_page": { "type": "integer" },
+            "per_page": { "type": "integer" },
+            "total_pages": { "type": "integer" },
+            "total_entries": { "type": "integer" }
+          }
+        }
+      }
+    },
+    "links": {
+      "type": "object",
+      "required": ["self", "first", "prev", "next", "last"],
+      "properties": {
+        "self": { "type": "string" },
+        "first": { "type": "string" },
+        "prev": { "type": ["string", "null"] },
+        "next": { "type": ["string", "null"] },
+        "last": { "type": "string" }
+      }
+    }
+  }
+}
```

Comment:
Please add newline at the end

Reply:
hmm very odd i tried, however on my local the extra line is already there so there is nothing to push according to my git. I have some related work for vets-api  that ill be pushing soon, Ill see if I can add this new line on that pr after this is merged.